ecc/Makefile

```diff
@@ -73,7 +73,7 @@ docker-fpc-app: docker-boilerplate-ecc
 		exit 1;\
 	fi
 	echo "\033[1;33mWARNING: overriding $(DOCKER_IMAGE) docker image\033[0m"
-	$(DOCKER) build $(DOCKER_BUILD_OPTS) -t $(DOCKER_IMAGE):$(FPC_VERSION) -f Dockerfile.fpc-app \
+	$(DOCKER) build $(DOCKER_BUILD_OPTS) -t $(DOCKER_IMAGE):$(FPC_VERSION) -t $(DOCKER_IMAGE):latest -f Dockerfile.fpc-app \
```

Comment:
That is pretty cool - I was not aware that you can just define multiple tags here in the command. 

We should use this in other places as well. Example: https://github.com/hyperledger/fabric-private-chaincode/blob/main/ecc_go/build.mk#L68

Reply:
you can't do  `-t $(DOCKER_IMAGE):$(FPC_VERSION) $(DOCKER_IMAGE):latest `because I tried and it didn't work so I tried adding the -t and it worked :-)